closed/OpenJ9.gmk

```diff
@@ -378,6 +378,12 @@ endif
 	@$(MKDIR) -p $(MODULES_LIBS_DIR)/java.base/server
 	@$(CP) -p $(OUTPUTDIR)/vm/redirector/$(LIBRARY_PREFIX)jvm_jdk11$(SHARED_LIBRARY_SUFFIX) $(MODULES_LIBS_DIR)/java.base/server/$(LIBRARY_PREFIX)jvm$(SHARED_LIBRARY_SUFFIX)
 
+ifeq (windows,$(OPENJDK_TARGET_OS))
+	@$(ECHO) "Updating vm/lib/jvm.* with vm/j9vm_jdk11/jdk11_jvm.*"
+	@$(CP) -p $(OUTPUTDIR)/vm/j9vm_jdk11/jdk11_jvm$(STATIC_LIBRARY_SUFFIX) $(OUTPUTDIR)/vm/lib/jvm$(STATIC_LIBRARY_SUFFIX)
+	@$(CP) -p $(OUTPUTDIR)/vm/j9vm_jdk11/jdk11_jvm.exp $(OUTPUTDIR)/vm/lib/jvm.exp
```

Comment:
Does it still work without copying the .exp file?

Reply:
Already recompiled and it seems there is no issue detected during compilation (also no problem with jdk/bin/java -version). So removed the setting for jvm.exp.